Valigator.TestApi.Core31/Controllers/TestController.cs

```diff
@@ -44,7 +44,7 @@ public GuidInnerClass(Guid identifier)
 	[ValigatorModel]
 	public class BodyClass
 	{
-		public Data<InnerClass[]> IdentifierCollection { get; set; } = Data.Collection<InnerClass>().Required().ItemCount(1);
+		public Data<InnerClass[]> IdentifierCollection { get; set; } = Data.Collection<InnerClass>().Required().ItemCount(1, 1);
```

Comment:
Makes for easier testing of the issue I was hunting down. I can revert this type of change if you are concerned with it.